cluster/swarm/cluster.go

```diff
@@ -823,32 +823,32 @@ func (c *Cluster) TotalCpus() int64 {
 // Info returns some info about the cluster, like nb or containers / images
 func (c *Cluster) Info() [][2]string {
 	info := [][2]string{
-		{"\bStrategy", c.scheduler.Strategy()},
-		{"\bFilters", c.scheduler.Filters()},
-		{"\bNodes", fmt.Sprintf("%d", len(c.engines)+len(c.pendingEngines))},
+		{"Strategy", c.scheduler.Strategy()},
+		{"Filters", c.scheduler.Filters()},
+		{"Nodes", fmt.Sprintf("%d", len(c.engines)+len(c.pendingEngines))},
 	}
 
 	engines := c.listEngines()
 	sort.Sort(cluster.EngineSorter(engines))
 
 	for _, engine := range engines {
-		info = append(info, [2]string{engine.Name, engine.Addr})
-		info = append(info, [2]string{" └ Status", engine.Status()})
-		info = append(info, [2]string{" └ Containers", fmt.Sprintf("%d", len(engine.Containers()))})
-		info = append(info, [2]string{" └ Reserved CPUs", fmt.Sprintf("%d / %d", engine.UsedCpus(), engine.TotalCpus())})
-		info = append(info, [2]string{" └ Reserved Memory", fmt.Sprintf("%s / %s", units.BytesSize(float64(engine.UsedMemory())), units.BytesSize(float64(engine.TotalMemory())))})
+		info = append(info, [2]string{" " + engine.Name, engine.Addr})
+		info = append(info, [2]string{"  └ Status", engine.Status()})
```

Comment:
@vieux since previously all `DriverStatus` output was printed with an extra space. So to maintain the formatting, we need to explicitly add that space. This is what it should look like:
<img width="511" alt="screen shot 2016-02-01 at 9 15 24 am" src="https://cloud.githubusercontent.com/assets/1872537/12724841/a3bb8088-c8c4-11e5-91cb-b8aca1ae0099.png">